ac_c.lua

```diff
@@ -239,6 +246,14 @@ for i, event in pairs(negativePayEvents) do
 	AddEventHandler(event, negativePayFunc)
 end
 
+AddEventHandler("onClientResourceStart", function(resourceName)
```

Comment:
can this trigger if a player is joining? i never used onClientResourceStart, also, would this trigger on an anticheese restart?

Reply:
This triggers when someone starts a resource from a client console, if I'm not wrong.

Reply:
So yes. this would also trigger when anticheese is being restarted (from the client)